scripts/update-changelog

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env node
+
+var fs = require('fs');
+
+var changelogFile = process.cwd() + '/CHANGELOG.md';
+var insertMarker = '<!--ENTRYINSERT-->';
+var versionMarker = ['<!--LATEST=', '-->'];
+var versionRegStr = '(\\d+)\\.(\\d+)\\.(\\d+)';
+var versionMarkerReg = new RegExp(versionMarker.join(versionRegStr));
+
+try {
+	var changelog = fs.readFileSync(changelogFile).toString();
+} catch(err) {
+	if (err.code === 'ENOENT') {
+		err.message += '. Make sure to run from sdk root directory'
+	}
+	throw err;
+}
+
+var latest = changelog.match(versionMarkerReg);
+var lmajor = +latest[1];
+var lminor = +latest[2];
+var lpatch = +latest[3];
+
+var version = process.argv[2];
+if (version !== undefined) {
+	if (version === 'major') {
+		version = (lmajor + 1) + '.0.0';
+	} else if (version === 'minor') {
+		version = lmajor + '.' + (lminor + 1) + '.0';
+	} else {
+		version = version.match(new RegExp('^' + versionRegStr + '$'));
+		if (version) {
+			// convert to num for comparison and for normalizing leading zeros
+			var major = +version[1];
```

Comment:
It works, but `parseInt(version[x], 10)` is clearer and doesn't make use of a JS quirk to convert a string to a number. We'll never need to worry about floats with our versioning.